### PR TITLE
[WIP] Add page for installing a new OS on a robot

### PR DESCRIPTION
**Pull request by Bidski**
[Preview](https://deploy-preview-27--nubook.netlify.com/guides/nubots/flashing)

### CODE REVIEW

File: src/book/02-projects/01-nubots/11-flashing.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Finally, once that script has finished, you must run one more command.
+
+```sh
+/mnt/arch-post_install.sh
+```
+
+This command will end by rebooting the robot. When this happens be sure to remove the USB installation drive from robot so that you may boot into the new system.
+
+The final two scripts can be found at [doc/ArchInstall/arch-chroot_install.sh](doc/ArchInstall/arch-chroot_install.sh) and [doc/ArchInstall/arch-post_install.sh](doc/ArchInstall/arch-post_install.sh)
```
Comment (ysims) [inline src/book/02-projects/01-nubots/11-flashing.mdx]: these links dont work

File: src/book/02-projects/01-nubots/11-flashing.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+title: Flashing a Robot
+description: How to installing a new operating system on the robot
+slug: /projects/nubots/flashing
+---
+
+## Flashing a Robot
+
+To install Arch Linux (our OS of choice) on to a robot perform the following instructions
+
+1. Download the latest [Arch Linux LiveUSB](https://www.archlinux.org/download/) image and burn it on to a [USB thumb drive](https://wiki.archlinux.org/index.php/USB_flash_installation_media#In_GNU/Linux)
```
Comment (ysims) [inline src/book/02-projects/01-nubots/11-flashing.mdx]: dd didn't work, suggest etcher

File: src/book/02-projects/01-nubots/11-flashing.mdx [hunk 3]
```diff
@@ -0,0 +1,169 @@
+---
+section: Projects
+chapter: NUbots
+title: Flashing a Robot
+description: How to installing a new operating system on the robot
```
Comment (JosephusPaye) [inline src/book/02-projects/01-nubots/11-flashing.mdx]:
```suggestion
description: How to install a new operating system on a robot.
```

File: src/book/02-projects/01-nubots/11-flashing.mdx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+     ![BIOS Power Menu](./images/flashing/bios_power.jpg 'BIOS Power Menu')
+   - Navigate to "Boot" and then "Boot Configuration"
+     - Ensure "Boot USB Devices First" is enabled in "UEFI Boot"
+     - Ensure "USB" is enabled in "Boot Devices"
+       ![BIOS Boot Configuration Menu](./images/flashing/bios_boot_configuration.jpg 'BIOS Boot Configuration Menu')
+   - Navigate to "Secure Boot" and disable "Secure Boot"
+     ![BIOS Secure Boot Menu](./images/flashing/bios_secure_boot.jpg 'BIOS Secure Boot Menu')
+   - Press F10 to save settings and exit and ensure you USB drive is plugged in
+
+1. If you are not using a brand new NUC, ensure your USB drive is plugged in and turn the NUC on
```
Comment (JosephusPaye) [inline src/book/02-projects/01-nubots/11-flashing.mdx]: Numbering:

```suggestion
2. If you are not using a brand new NUC, ensure your USB drive is plugged in and turn the NUC on
```

Same with 3, 4 and 5 below, as well as in **Manually setting up a WiFi connection on the robot**.

Or is the numbering messed up when you format the file?

**Comment by JosephusPaye** [comment]
Content generally LGTM. Was @ysims able to verify that all the instructions work?

**Comment by ysims** [comment]
Waiting for NUbots/NUbots#274 so I can test this.

**Comment by Bidski** [comment]
@JosephusPaye does your review still stand or can we dismiss it?

**Pull request merged**